servant/src/Servant/API/UVerb.hs

```diff
@@ -71,18 +73,8 @@ instance (HasStatus a, HasStatuses as) => HasStatuses (a ': as) where
   statuses _ = statusOf (Proxy :: Proxy a) : statuses (Proxy :: Proxy as)
 
 newtype WithStatus (k :: Nat) a = WithStatus a
-  deriving (Eq, Show, GHC.Generic)
-
-instance (GHC.Generic (WithStatus n a), ToJSON a) => ToJSON (WithStatus n a)
-
-instance (GHC.Generic (WithStatus n a), FromJSON a) => FromJSON (WithStatus n a)
-
-instance MimeRender ctype a => MimeRender ctype (WithStatus _status a) where
-  mimeRender contentTypeProxy (WithStatus a) = mimeRender contentTypeProxy a
-
-instance MimeUnrender ctype a => MimeUnrender ctype (WithStatus _status a) where
-  mimeUnrender contentTypeProxy input =
-    WithStatus <$> mimeUnrender contentTypeProxy input
+  deriving stock (Eq, Show, Generic)
```

Comment:
this may fail on some old GHC versions we try to support for some reason :)

Reply:
Oh yeh this will fail on `8.0` indeed. Let me remove it for now.